### PR TITLE
Update TaskRouter javadoc

### PR DESCRIPTION
**Pull request by skimbrel**

### CODE REVIEW

File: src/main/java/com/twilio/sdk/resource/instance/taskrouter/Task.java [hunk 1]
```diff
@@ -124,17 +126,8 @@ public int getPriority() {
 		}
 	}
 
-    /**
-     * Gets the timeout.
-     *
-     * @return the timeout
-     */
-    public int getTimeout() {
```
Comment (AlexPayment) [inline src/main/java/com/twilio/sdk/resource/instance/taskrouter/Task.java]: This property is still used by TaskTest.java

**Comment by skimbrel** [comment]
that would explain the build failure. I might have removed it incorrectly… it doesn't show up in the docs but is it in the response?

File: src/main/java/com/twilio/sdk/resource/instance/taskrouter/Task.java (hunk 1 above)
Comment (AlexPayment) [inline src/main/java/com/twilio/sdk/resource/instance/taskrouter/Task.java]: Yes, it's in the response.

File: src/main/java/com/twilio/sdk/resource/instance/taskrouter/Event.java [hunk 2]
```diff
@@ -58,7 +63,7 @@ public String getAccountSid() {
     }
 
     /**
-     * Gets the actor sid.
+     * The unique ID of the actor that generated this Event.
```
Comment (AlexPayment) [inline src/main/java/com/twilio/sdk/resource/instance/taskrouter/Event.java]: Doesn't seem to be in the taskrouter doc

File: src/main/java/com/twilio/sdk/resource/instance/taskrouter/Event.java [hunk 3]
```diff
@@ -67,7 +72,7 @@ public String getActorSid() {
     }
 
     /**
-     * Gets the actor type.
+     * The type of the actor that generated this Event.
```
Comment (AlexPayment) [inline src/main/java/com/twilio/sdk/resource/instance/taskrouter/Event.java]: Doesn't seem to be in the taskrouter doc

File: src/main/java/com/twilio/sdk/resource/instance/taskrouter/Event.java [hunk 4]
```diff
@@ -76,7 +81,7 @@ public String getActorType() {
     }
 
     /**
-     * Gets the actor url.
+     * The URL for the resource representing the actor that generated this Event.
```
Comment (AlexPayment) [inline src/main/java/com/twilio/sdk/resource/instance/taskrouter/Event.java]: Doesn't seem to be in the taskrouter doc

Comment (skimbrel) [inline src/main/java/com/twilio/sdk/resource/instance/taskrouter/Event.java]: I went and generated an event and these fields are all in it. ¯\_(ツ)_/¯

File: src/main/java/com/twilio/sdk/resource/instance/taskrouter/Task.java [hunk 5]
```diff
@@ -124,17 +126,16 @@ public int getPriority() {
...
-        return (Integer)getObject("timeout");
-    }
+	/**
+	 * The Task assignment request timeout, in seconds.
+	 * @return the timeout
```
Comment (AlexPayment) [inline src/main/java/com/twilio/sdk/resource/instance/taskrouter/Task.java]: Nitpick: a space before this line

**Comment by AlexPayment** [comment]
The property Reason is missing from Task.

File: src/main/java/com/twilio/sdk/resource/instance/taskrouter/Workspace.java [hunk 6]
```diff
@@ -115,7 +115,7 @@ public String getDefaultActivitySid() {
 	}
 
 	/**
-	 * Retrieves a {@link com.twilio.sdk.resource.instance.taskrouter.Event} from a Workspace
+	 * Retrieves a {@link com.twilio.sdk.resource.instance.taskrouter.Event} from this Workspace
```
Comment (AlexPayment) [inline src/main/java/com/twilio/sdk/resource/instance/taskrouter/Workspace.java]: a -> an

**Comment by skimbrel** [comment]
Test failure didn't reproduce. Ready for merge, @AlexPayment?

**Pull request merged**